twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -185,15 +185,13 @@ def _compile_scala(ctx, label, output, scrooge_jar, deps_java_info,
       expect_java_output = False,
       scalac_jvm_flags = [])
 
-  return java_common.create_provider(
-      use_ijar = False,
-      source_jars = [scrooge_jar],
-      compile_time_jars = depset(
-          [output], transitive = [merged_deps.compile_jars]),
-      transitive_compile_time_jars = depset(
-          [output], transitive = [merged_deps.transitive_compile_time_jars]),
-      transitive_runtime_jars = depset(
-          [output], transitive = [merged_deps.transitive_runtime_jars]))
+  return JavaInfo(
+      source_jar = scrooge_jar,
+      deps = deps_java_info + implicit_deps,
+      runtime_deps = deps_java_info + implicit_deps,
+      exports = deps_java_info + implicit_deps,
+      output_jar = output,
+      compile_jar = output)
```

Comment:
Just making sure you intentionally don’t want ijar here?

Reply:
See this comment on line 164:
```
   # this only compiles scala, not the ijar, but we don't
   # want the ijar for generated code anyway: any change	   
   # in the thrift generally will change the interface and	   
   # method bodies	   
```